production/helm/loki/templates/gateway/deployment-gateway-nginx.yaml

```diff
@@ -31,9 +31,6 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include "loki.configMapOrSecretContentHash" (dict "ctx" . "name" "/gateway/configmap-gateway.yaml") }}
-        {{- with .Values.loki.podAnnotations }}
```

Comment:
as long as we're sure this doesn't negatively affect enterprise deployments, then I'm good with it